### DISCUSSION TITLE
Authentication mechanism to avoid spam / junk

### DISCUSSION THREAD

**Discussion by raymcdermott** [repl-acement/repl-acement#27] [Ideas]
Ideally link the user to a key (eg keybase.io) and have a web of trust

**Comment by raymcdermott** [comment]
@yenda is on the case

**Comment by raymcdermott** [comment]
Who knew that keybase had an API

https://keybase.io/docs/api/1.0

**Comment by raymcdermott** [comment]
OTOH https://keybase.io/blog/keybase-joins-zoom

**Comment by andrewzhurov** [comment]
How such spam/junk would look like?
I don't have my head wrapped around the meant architecture just yet

**Reply by raymcdermott** [reply]
we don't want random updates to eg `clojure.core` from outside of canonical sources. There has to be some way to establish trust for the writers.
